routes/htmlRoutes.js

```diff
@@ -2,22 +2,13 @@ var db = require("../models");
 
 module.exports = function(app) {
   // Load index page
-  app.get("/", function(req, res) {
-    db.Example.findAll({}).then(function(dbExamples) {
-      res.render("index", {
-        msg: "Welcome!",
-        examples: dbExamples
-      });
-    });
+  app.get("/Launch.html", function(req, res) {
```

Comment:
@jayxmiller24 you don't need the .html here.
```suggestion
  app.get("/launch", function(req, res) {
```